### PR TITLE
Add sexyLoot

### PR DESCRIPTION
**Pull request by sexyLoot**
<!-- 
Please use this pull request template when adding a new resource to the website. 
You do not have to fill out all the fields, but please attempt to complete as many as possible.

sexyLoot sLoots

Etherscan link: https://etherscan.io/address/0xb13EFaB167751A95EC6F9844944C74Aa668ddeD6#writeContract

Website: https://sexyloot.co/

Twitter: https://twitter.com/LootSexy

### CODE REVIEW

**Comment by sexyLoot** [comment]
Followed the instructions from here
https://github.com/lootproject/website/pull/70
But now not sure what "checks have failed" means?

File: utils/lists.ts [hunk 1]
```diff
@@ -442,6 +442,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Mysterious runes randomly generated and carved on chain. Gift from ancient world!",
     url: "https://runesmystery.github.io/"
   },
+ {
+    name: “sexy Loot - sLoots“,
```
Comment (ravindranrahul) [inline utils/lists.ts:446]: Please remove the double quotes and type them manually or copy what you done with the url

File: utils/lists.ts [hunk 2]
```diff
@@ -442,6 +442,11 @@ export const derivativesList: Record<string, string>[] = [
...
+    description: “sLoots are on-chain harlots and strumpets.”,
```
Comment (ravindranrahul) [inline utils/lists.ts:447]: Update the quotes here as well.

**Review by ravindranrahul — CHANGES_REQUESTED** [review]

File: utils/lists.ts (hunk 2 above)
Comment (sexyLoot) [reply utils/lists.ts:447]: Ah darn it. Got it.

**Pull request closed without merging**